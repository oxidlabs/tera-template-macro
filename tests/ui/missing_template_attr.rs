use tera_template_macro::TeraTemplate;

#[derive(TeraTemplate, serde::Serialize)]
struct Smoke {
    hello: String
}

fn main() {}