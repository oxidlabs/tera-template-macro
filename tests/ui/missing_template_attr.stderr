error: The #[template(path = "...")] is required.
 --> tests/ui/missing_template_attr.rs:4:1
  |
4 | / struct Smoke {
5 | |     hello: String
6 | | }
  | |_^
